src/EditorFeatures/Test/CodeFixes/CodeFixServiceTests.cs

```diff
@@ -1079,7 +1088,7 @@ void M()
         await diagnosticIncrementalAnalyzer.GetDiagnosticsForIdsAsync(
             sourceDocument.Project.Solution, sourceDocument.Project.Id, sourceDocument.Id, diagnosticIds: null, shouldIncludeAnalyzer: null, getDocuments: null,
             includeLocalDocumentDiagnostics: true, includeNonLocalDocumentDiagnostics: true, CancellationToken.None);
-        await diagnosticIncrementalAnalyzer.GetTestAccessor().TextDocumentOpenAsync(sourceDocument);
+        // await diagnosticIncrementalAnalyzer.GetTestAccessor().TextDocumentOpenAsync(sourceDocument);
```

Comment:
broken test.  it was manipulating internal state in a way that normal product code could not.